sources/core/Stride.Core.Design/IO/UPath.cs

```diff
@@ -292,7 +292,7 @@ public override string ToString()
         [NotNull]
         public string ToWindowsPath()
         {
-            return FullPath.Replace('/', '\\');
+            return OperatingSystem.IsWindows() ? FullPath.Replace('/', '\\') : FullPath;
```

Comment:
The method is explicitly named `ToWindowsPath` so it should always return a windows path on any platform.